rundeckapp/grails-app/services/rundeck/services/ScheduledExecutionService.groovy

```diff
@@ -3277,6 +3277,15 @@ class ScheduledExecutionService implements ApplicationContextAware, Initializing
      */
     public void jobDefinitionBasic(ScheduledExecution scheduledExecution, ScheduledExecution input, Map params, UserAndRoles userAndRoles) {
         Map basicProps
+        if (params.jobDetailsJson) {
+            def detailsData = JSON.parse(params.jobDetailsJson.toString())
+
+            if(detailsData instanceof JSONObject) {
+                detailsData.keySet().forEach(detailKey -> {
+                    scheduledExecution[detailKey as String] = detailsData.get(detailKey)
```

Comment:
this should add data to the basicProps Map, later the scheduledExecution is updated via the basicProps. 
since the basicProps is not initialized yet, you could move this after line 3313